consensus/istanbul/core/roundchange.go

```diff
@@ -170,3 +251,21 @@ func (rcs *roundChangeSet) MaxRound(num int) *big.Int {
 	}
 	return maxRound
 }
+
+func (rcs *roundChangeSet) getCertificate(r *big.Int, f int) (istanbul.RoundChangeCertificate, error) {
+	rcs.mu.Lock()
+	defer rcs.mu.Unlock()
+
+	round := r.Uint64()
+	if rcs.roundChanges[round].Size() > 2*f {
```

Comment:
I think that this `2F` comes from PBFT directly. The IBFT correctness paper does not touch on this, but I'd like to double check if this should be a quorum instead.